app/services/ContributionsFrontendService.scala

```diff
@@ -30,13 +30,18 @@ class ContributionsFrontendService(wsClient: WSClient, contributionsFrontendUrl:
     }
   }
 
-  def execute(queryString: Map[String, Seq[String]])(implicit req: RequestHeader, ec: ExecutionContext): EitherT[Future, Throwable, Option[Email]] = {
+  def execute(request: Request[AnyContent])(implicit req: RequestHeader, ec: ExecutionContext): EitherT[Future, Throwable, Option[Email]] = {
     import cats.syntax.applicativeError._
     import cats.instances.future._
+
+    val wcCookies = request.cookies.toSeq.map { c =>
+      DefaultWSCookie(c.name, c.value, c.domain, Some(c.path), c.maxAge.map(_.toLong), c.secure, c.httpOnly)
+    }
     val endpoint = "/paypal/uk/execute"
     wsClient.url(s"$contributionsFrontendUrl/$endpoint")
-      .withHttpHeaders("Content-Type" -> "application/json")
-      .withQueryStringParameters(convertQueryString(queryString): _*)
+      .withQueryStringParameters(convertQueryString(request.queryString): _*)
```

Comment:
Yep good shout taking out the `Content-Type` - there is no payload body since this is a GET request. (If it were, say, a POST request with a JSON body then you want want to leave this in: https://tools.ietf.org/html/rfc7231#section-3.1.1.5)